v2/client/client.go

```diff
@@ -246,13 +246,9 @@ func (c *ceClient) StartReceiver(ctx context.Context, fn interface{}) error {
 				}
 
 				// Do not block on the invoker.
-				wg.Add(1)
```

Comment:
This change was introduced in #574 to solve #569, But I think this change was not required to solve #569

Reply:
this change makes the invoker single threaded, and is required to solve #569